src/Components/Modal/ModalWrapper.tsx

```diff
@@ -155,20 +161,28 @@ export const ModalContainer = styled.div.attrs<{
   box-shadow: 0 4px 15px rgba(0, 0, 0, 0.15);
   animation: ${slideUp} 250ms linear;
 
-  ${media.sm`
-    width: 100%;
-    border-radius: 0;
-  `};
+  ${props =>
+    props.isResponsive
+      ? media.sm`
+          width: calc(100vw - 20px);
```

Comment:
`calc` always seems like a bit of a smell to me, but I think we need an explicit width here for centering that `ModalContainer` applies.

Reply:
Interesting- why is it a smell for you? In all cases or just css in js?

Reply:
I think this is actually a nice use of `calc` :+1:

Reply:
I find `calc` to be brittle over time, like here it's not clear _why_ `20px` is being subtracted, outside of the comment on this PR, though I guess I could / should add a comment inline here. Also, it's less of an issue now, but `calc` used to have slightly different implementations and constraints across browsers that often led to quirks. My preference for a modal layout like this would be to use flex box for the layout.